### ISSUE TITLE
Quest: Free at Last

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#27]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.3

Thousand Needles / Darkcloud Pinnacle / NPC: Lakota Windsong / Quest: Free at Last
Link: https://www.wowhead.com/wotlk/quest=4904/free-at-last

Description: playback wrong voice-over.

Completion: playback wrong voice-over.

**Comment by carevass** [comment]
Will be fixed in WoTLK v1.1, there is a similarly named quest that makes conflict with this one

**Comment by carevass** [comment]
This should be fixed with new table in AI_VoiceOverData_WoTLK_v1.1.
